Doc/using/venv-create.inc

```diff
@@ -83,6 +83,17 @@ The command, if run with ``-h``, will show the available options::
    particular note is that double-clicking ``python.exe`` in File Explorer
    will resolve the symlink eagerly and ignore the virtual environment.
 
+.. note::
+   On Microsoft Windows, it may be required to enable the ``Activate.ps1``
+   script by setting the execution policy for the user. You can do this by
+   issuing the following Powershell command from a Powershell command window:
+
+   PS C:\> Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser
+
+   See `About Execution Policies
+   <https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_execution_policies?view=powershell-7>`_
```

Comment:
Is there a more generic redirection link for this? Otherwise this is looks pinned to PowerShell 7 which will become obsolete at some point.
```suggestion
   <https:/go.microsoft.com/fwlink/?LinkID=135170>`_
```

Reply:
:heavy_check_mark: